### PR TITLE
MAL-248 | OR operator

### PR DESCRIPTION
**Pull request by Mitsuroseba**

### CODE REVIEW

**Comment by naebal** [comment]
Build comment file: 
--------------
===============
Build site installed at http://95.85.26.139/build29
Behat tests: standard file http://95.85.26.139/build29/build_reports/report.html

--------------

Refer to this link for build results (access rights to CI server needed): 
http://95.85.26.139:8080//job/malmo_PR_BUILDER/29/
Test PASSed.

File: drupal/profiles/mal/modules/custom/mal_update/mal_update.install [hunk 1]
```diff
@@ -908,3 +908,13 @@ function mal_update_update_7068() {
...
   module_enable(array('page_user_pages', 'user_pages'));
 }
+
+/**
+ * OR operator for course periods.
+ */
+function mal_update_update_7070() {
+  module_enable(array('search_api_ranges'));
+  features_revert_module('config_facet');
+  module_disable(array('mal_facetapi', 'facetapi_ranges'));
```
Comment (M1r1k) [inline drupal/profiles/mal/modules/custom/mal_update/mal_update.install]: If you disable module mal_facetapi why do you have so much modifications there?

**Comment by naebal** [comment]
Build comment file: 
--------------
===============
Build site installed at http://95.85.26.139/build30
Behat tests: standard file http://95.85.26.139/build30/build_reports/report.html

--------------

Refer to this link for build results (access rights to CI server needed): 
http://95.85.26.139:8080//job/malmo_PR_BUILDER/30/
Test PASSed.

**Comment by naebal** [comment]
Build comment file: 
--------------
===============
Build site installed at http://95.85.26.139/build31
Behat tests: standard file http://95.85.26.139/build31/build_reports/report.html

--------------

Refer to this link for build results (access rights to CI server needed): 
http://95.85.26.139:8080//job/malmo_PR_BUILDER/31/
Test PASSed.

**Pull request merged**